src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -286,6 +337,14 @@
         sectionSelect.addEventListener('change', toggleDisabledStateOnApplyBtn, false);
     }
 
+    for (const tagType in clearSearchTagBtnMethods) {
+        const tagBtns = doc.querySelectorAll(`.ez-tag__remove-btn--${tagType}`);
+
+        if (tagBtns.length) {
```

Comment:
`if` is unnecessary, `querySelectorAll` will always return array and `forEach` can be executed on empty array